<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1708956281537' stop='1708956295438'>
    <name>PTRF</name>
    <title>PTRF</title>
    <test-cases>
        <test-case start='1708956281538' status='failed' stop='1708956295427'>
            <name>CT09-Resumo_dos_Recursos-TodasContas</name>
            <title>CT09-Resumo_dos_Recursos-TodasContas</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Timed out retrying after 4000ms: Expected to find element: `.row > :nth-child(2) > .pt-0 > :nth-child(1)`, but never found it.</message>
                <stack-trace>AssertionError: Timed out retrying after 4000ms: Expected to find element: `.row > :nth-child(2) > .pt-0 > :nth-child(1)`, but never found it.
                    at ResumoPagina.selecionarTipoCheque (webpack://cypress-ptrf/./cypress/support/Paginas/ResumoPagina.js:32:63)
                    at Context.eval (webpack://cypress-ptrf/./cypress/e2e/PTRF/CT11-Resumo_dos_Recursos-TipoCheque.cy.js:32:11)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>